R/import_mzroll.R

```diff
@@ -357,15 +381,20 @@ create_sqlite_con <- function(sqlite_path) {
 #'   peak-level data.
 #'
 #' @param mzroll_db_con an SQLite connection to an mzrollDB database
-#'
+#' @param quant_col name of quant column to use for standardization from the
+#'   peaks table.
+#' \code{default = "peakAreaTop"}
+#' 
 #' @return a table of peakgroups
-process_mzroll_load_peakgroups <- function(mzroll_db_con) {
+process_mzroll_load_peakgroups <- function(mzroll_db_con, quant_col = "peakAreaTop") {
   peakgroup_positions <- dplyr::tbl(mzroll_db_con, "peaks") %>%
     dplyr::collect() %>%
     dplyr::group_by(groupId) %>%
+    dplyr::mutate(quant_value := !! sym(quant_col)) %>%
```

Comment:
I don't think you need the := since you are creating a variable named "quant_value" rather than to variable defined by a quosure

Reply:
fixed